test/featureUtils_unit_test.js

```diff
@@ -6,34 +6,49 @@ import Coordinates from '../src/Core/Geographic/Coordinates';
 const assert = require('assert');
 const geojson = require('../examples/geojson/simple.geojson.json');
 
-const feature = GeoJSON2Features.parse('EPSG:4326', geojson, undefined, { buildExtent: true });
+const promise = GeoJSON2Features.parse('EPSG:4326', geojson, undefined, { buildExtent: true });
 
 describe('FeaturesUtils', function () {
-    it('should correctly parse geojson', () => {
-        assert.equal(feature.geometries.length, 3);
+    it('should correctly parse geojson', (done) => {
+        promise.then((feature) => {
```

Comment:
With newer version of `mocha`, you can directly return a promise, this will have the same effect than the `done` mechanism.

Reply:
But actually, GeoJSON2Features could return a plain object, see above.

Reply:
> With newer version of mocha

Does our version of `mocha` support this?

Reply:
Yes, I did it for the `getCapabilities` PR